stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -112,6 +136,30 @@ internal class PaymentOptionsActivity : BasePaymentSheetActivity<PaymentOptionRe
         }
     }
 
+    private fun setupPrimaryButton() {
+        // TODO: implement
+    }
+
+    private fun onTransitionTarget(
+        transitionTarget: PaymentOptionsViewModel.TransitionTarget,
+        fragmentArgs: Bundle
+    ) {
+        supportFragmentManager.commit {
+            when (transitionTarget) {
+                PaymentOptionsViewModel.TransitionTarget.AddPaymentMethodFull -> {
+                    // TODO: implement
+                }
+                PaymentOptionsViewModel.TransitionTarget.SelectSavedPaymentMethod -> {
+                    // TODO: implement
```

Comment:
did we standardize on TODO(name) format? doesn't matter just curious

Reply:
Yeah, should have added username